packages/datatrak-web/src/features/SurveyQuestion.tsx

```diff
@@ -0,0 +1,104 @@
+// @ts-nocheck
```

Comment:
This looks like a old test file which can be deleted. The actual SurveyQuestion component that is used in the SurveyScreen is in the features/Survey folder